From 01f4679c5a5db804fbdcb3348a470f9b65e2993f Mon Sep 17 00:00:00 2001
From: Jyri Sarha <jsarha@ti.com>
Date: Mon, 12 Sep 2016 18:45:30 +0300
Subject: [PATCH 077/895] drm/tilcdc: Remove "default" from
 blue-and-red-wiring property binding

Remove "default" keyword from blue-and-red-wiring devicetree property
binding document. The code does not support and there is no intention
to support it.

Reported-by: Rob Herring <robh@kernel.org>
Signed-off-by: Jyri Sarha <jsarha@ti.com>
(cherry picked from commit 34355ee0d5480f0f437db4d418bd6c494883f708)
---
 .../devicetree/bindings/display/tilcdc/tilcdc.txt  |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/tilcdc/tilcdc.txt b/Documentation/devicetree/bindings/display/tilcdc/tilcdc.txt
index a5007aa..a83abd7 100644
--- a/Documentation/devicetree/bindings/display/tilcdc/tilcdc.txt
+++ b/Documentation/devicetree/bindings/display/tilcdc/tilcdc.txt
@@ -17,9 +17,9 @@ Optional properties:
    the lcd controller.
  - max-pixelclock: The maximum pixel clock that can be supported
    by the lcd controller in KHz.
- - blue-and-red-wiring: Recognized values "default", "straight" or
-   "crossed". This property deals with the LCDC revision 2 (found on
-   AM335x) color errata [1].
+ - blue-and-red-wiring: Recognized values "straight" or "crossed".
+   This property deals with the LCDC revision 2 (found on AM335x)
+   color errata [1].
     - "straight" indicates normal wiring that supports RGB565,
       BGR888, and XBGR8888 color formats.
     - "crossed" indicates wiring that has blue and red wires
-- 
1.7.5.4

